packages/astro/src/core/errors/overlay.ts

```diff
@@ -445,6 +481,27 @@ class ErrorOverlay extends HTMLElement {
 		this.root = this.attachShadow({ mode: 'open' });
 		this.root.innerHTML = overlayTemplate;
 
+		// theme toggle logic
+		const themeToggleButton = this.root.querySelector<HTMLButtonElement>('#theme-toggle');
+		if (
+			localStorage.astroErrorOverlayTheme === 'dark' ||
```

Comment:
Not sure if it would be an issue but, I made the name of the key verbose so that it'd be unlikely to coincide with a key the developer has set in their main site.